website/docs/reference/global-configs/about-global-configs.md

```diff
@@ -8,4 +8,11 @@ Global configs enable you to fine-tune _how_ dbt runs projects on your machine
 
 Global configs control things like the visual output of logs, the manner in which dbt parses your project, and what to do when dbt finds a version mismatch or a failing model. These configs are "global" because they are available for all dbt commands, and because they can be set for all projects running on the same machine or in the same environment.
 
-Starting in v1.0, you can set global configs in three places. When all three are set, command line flags take precedence, then environment variables, and last yaml configs (usually `profiles.yml`).
\ No newline at end of file
+### Global config precedence
+
+Starting in v1.0, you can set global configs in three places. dbt will pick the config in the following order (lower takes priority):
```

Comment:
Yeah, it would be good to choose an ordering that makes sense! My brain works from top-to-bottom, so I think I'd actually prefer flipping the order. But I chose this order because it best aligned with other examples I'd seen within our documentation.

Applied code suggestions and opened this to review this more comprehensively:
- https://github.com/dbt-labs/docs.getdbt.com/issues/4371